praisonai_tools/tools/joy_trust_tool.py

```
@@ -147,80 +156,112 @@ def check_trust(self, agent_name: str, min_score: Optional[float] = None) -> Dic
                 "threshold_used": min_threshold,
-                "reputation": {},
-                "recommendations": 0,
+                "vouch_count": 0,
+                "capabilities": [],
+                "tier": None,
+                "badges": [],
                 "error": f"API error ({e.response.status_code}): {e.response.text}",
                 "fallback_used": self.config.fallback_on_error
             }
-            return error_result
         except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)